exo.core.component.organization.api/src/main/java/org/exoplatform/services/organization/auth/SecurityCheckAuthenticationPlugin.java

```diff
@@ -0,0 +1,29 @@
+package org.exoplatform.services.organization.auth;
+
+import org.exoplatform.container.component.BaseComponentPlugin;
+import org.exoplatform.services.organization.User;
+
+public abstract class SecurityCheckAuthenticationPlugin extends BaseComponentPlugin {
```

Comment:
Why adding this plugin in `core` project level if it's not used inside, neither referenced? In addition, you miss the Header license in fact.

Reply:
@boubaker , I followed the same replacement location with the Authenticaterplugin, its' also  not used inside the core module, neither referenced

Reply:
Not same in fact, it's used here: https://github.com/Meeds-io/core/commit/9953ec25b8427bb02c87d3427b7600640ad989ab#diff-9b007092c59894b68a04cc614b92801bbdb4ab92cdf2dba7ec60b4b133d8d844R76

Reply:
header added : https://github.com/Meeds-io/core/commit/ceced312a7048b228e1b86b9d28c3bd33efae30d#diff-174ccd71867b79cfb62756ad4723a3c0d14c4805309f8ae6a625e65e36cdf4bfR1-R16

Ok i see, where do you suggest to add it ?

Reply:
Ok, i will proceed to do this, i will add it in gaten-portal

Reply:
@boubaker added in this PR: https://github.com/Meeds-io/gatein-portal/pull/379